CHANGES.md

```diff
@@ -1,14 +1,114 @@
 # Imath Release Notes
 
-## Inherited History from OpenEXR
-History dated before May 9th 2020 has been inherited from OpenEXR
-This history has been filtered using the tool [git-filter-repo](https://github.com/newren/git-filter-repo)
+* [Version 3.0.0-beta](#version-300-beta-Mar-15-2021) March 15, 2021
+* [Inherited History from OpenEXR](#inherited-history-from-openexr)
+
+## Version 3.0.0-beta (March 15, 2021)
+
+First release of Imath independent of OpenEXR.
 
-This tool can selectively remove unneeded history based on the directories to which commits have been made. For instance, it can remove all history from a given directory using the command:\
+See the [porting guide](docs/PortingGuide2-3.md) for details about
+differences from previous releases.
 
-git filter-repo --path old_dir1/ --path old_dir2/ --invert-paths\
+Summary:
 
-Or it can keep all history from certain paths using \
+* Imath includes the half type, formerly in a separate Half library.
+* Headers are installed in ``Imath/`` subdirectory.
+* All appropriate methods are marked constexpr, noexcept
+* Appropriate declaration include CUDA ``__host__`` and ``__device__``
+  directives.
+* Throwing methods throw std exceptions instead of ``Iex``.
+* New Vec and Matrix interoperability constructors for conversion from
+  other similar type objects.
+* Symbol linkage visibility is limited to specific public symbols.
+* python bindings are off by default, available by setting ``PYTHON=ON``.
+* Deprecated features:
+  - ``std::numeric_limits`` replaces ``Imath::limits``.
+  - ``Int64`` and ``SInt64`` are deprecated in favor of ``uint64_t``
+    and ``int64_t``.
 
-git filter-repo --path remaining_dir1/ --path remaining_dir2/
+### Merged Pull Requests
+
+* \[[#119](https://github.com/AcademySoftwareFoundation/Imath/pull/119)\] Enable policy 77 if possible.
```

Comment:
I noticed these escapes (  \[ \] ) have been removed in the OpenEXR release notes